src/marqo/core/structured_vespa_index/structured_vespa_index.py

```diff
@@ -480,7 +478,7 @@ def _to_vespa_lexical_query(self, marqo_query: MarqoLexicalQuery) -> Dict[str, A
             query_inputs.update(score_modifiers)
 
         query = {
-            'yql': f'select {select_attributes} from {self._marqo_index.schema_name} where {lexical_term}{filter_term}',
+            'yql': f'select {select_attributes} from {self._marqo_index.schema_name} where ({lexical_term}){filter_term}',
```

Comment:
This is fixed in @RaynorChavez's PR.

Reply:
Yep, we'll have to fix merge conflicts later